modules/vpc/variables.tf

```diff
@@ -0,0 +1,98 @@
+variable "tags" {
+  description = "A mapping of tags to assign to the resources"
+  type        = map(string)
+  default = {
+    terraform = true
+  }
+}
+
+variable "name" {
+  description = "The application name"
+  type        = string
+}
+
+variable "cidr" {
+  description = "The CIDR block for the vpc"
+  type = string
+}
+
+variable "public_subnets" {
+  description = "A list of public subnets inside the VPC"
+  type        = list(string)
+}
+
+variable "private_subnets" {
+  description = "A list of private subnets inside the VPC"
+  type        = list(string)
+}
+
+variable "database_subnets" {
+  description = "A list of database subnets"
+  type        = list(string)
+}
+
+variable "azs" {
+  description = "A list of availability zones names or ids in the region"
+  type = list(string)
+}
+
+variable "create_database_subnet_group" {
+  type    = bool
+  default = true
+}
+
+variable "enable_nat_gateway" {
+  type    = bool
+  default = true
+}
+
+variable "single_nat_gateway" {
+  type    = bool
+  default = false
+}
+
+variable "enable_vpn_gateway" {
+  type    = bool
+  default = false
+}
+
+variable "enable_s3_endpoint" {
+  type    = bool
+  default = false
+}
+
+variable "enable_dhcp_options" {
+  type    = bool
+  default = false
```

Comment:
why did we default this to false when we're setting it to true everywhere?

Reply:
My initial thought for this was that I would set the default to how the terraform-aws-modules/vpc/aws module was setting them (figured it would be more consistent). It doesn't look like we're setting this to true in general for our vpcs so I kept it default false for our nerves hub vpc (in nerves_hub_infrastructure) and added the variable to app/vpc.tf in this repo since originally it was set to true.